src/logic/calculate.test.js

```diff
@@ -0,0 +1,12 @@
+import calculate from './calculate';
+
+describe('Calculate correctly', () => {
+  test('AC should clear the total, operation and next', () => {
+    const obj = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+    expect(calculate(obj, 'AC')).toEqual(obj);
+  });
+});
```

Comment:
- [ ] As a requirement all the functionalities & scenarios of the `operate.js` and `calculate.js` should be covered. Please kindly add more tests to your app 👍🏼